Formula/rbw.rb

```diff
@@ -0,0 +1,18 @@
+class Rbw < Formula
+  desc "Unofficial command-line client for Bitwarden"
+  homepage "https://github.com/doy/rbw"
+  url "https://github.com/doy/rbw/archive/0.5.2.tar.gz"
+  sha256 "5ab3c569ea55eeee6e70130c3d1a3399ac0529b666e2ba12264c1a7b0b5dbc3c"
+  license "MIT"
+
+  depends_on "rust" => :build
+  depends_on "pinentry"
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system "#{bin}/rbw", "--version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Since this is client that sync with an online remote server, don't how I could test more than that. I can eventually test that I can set a configuration value (`rbw config set item value`), but that don't test a lot more the core functionality of this tool. Everything else will try to connect to connect to a remote, or unlock an already existing database.

Do you have suggestions for better tests in this case?